src/SimpleSimulator.cpp

```diff
@@ -632,5 +696,4 @@ int main(int argc, char **argv) {
 
 
     return 0;
-}
-
+}
```

Comment:
You should add a newline at end of file.

Reply:
> You'll find my comments inline. This looks very good!

Thanks for the comments, @HerrHorizontal ! I'll work on them tomorrow :)

Reply:
Done: https://github.com/HEPCompSim/DCSim/pull/41/commits/cf56615acb2bb93093dcc80b3a6fc64408af4013